README.md

```diff
@@ -1,2 +1,39 @@
 # Decode-Morse-Code
 In this repository I will be creating a program using the Ruby Language to decode Morse code into English.
+
+![](https://img.shields.io/badge/Decode-Morse-Code-blueviolet)
+
+## Built With
+
+- Ruby
+
```

Comment:
- [ ] Let's make the readme file more descriptive by adding information on how users would go about cloning and running the project on their local machine.